static/bidder-info/missena.yaml

```diff
@@ -0,0 +1,16 @@
+endpoint: https://bid.missena.io/
+maintainer:
+  email: prebid@missena.com
+gvlVendorID: 687
+modifyingVastXmlAllowed: true
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+  site:
+    mediaTypes:
+      - banner
+userSync:
+  redirect:
+    url: https://sync.missena.io/iframe?gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}
+    userMacro: $UID
```

Comment:
This does not appear to be working. Two things:
1) I see `iframe` in your URL which leads me to believe your user sync is of type iframe instead of redirect. If that's the case, you should indicate as such:
```
userSync:
  iframe:
    url: ...
    userMacro: $UID
```

2) When I initiate a user sync I do not see the redirect to the PBS setuid endpoint, which will be specified in the redirect portion of your user sync query string (`redirect={{.RedirectURL}}`). This is where your user ID is set on the PBS cookie.